README.Rmd

```diff
@@ -40,78 +41,149 @@ devtools::install_github("ropensci/mregions")
 
-```{r}
-res1 <- mr_geojson(key = "Morocco:dam")
+Get Data
+
+```{r, eval=FALSE}
+ecoregions_geoJSON <- mr_geojson(key = "Ecoregions:ecoregions")
+# check number of features
+length(ecoregions_geoJSON$features)
 ```
 
-Get helper library
+Plot Data
 
 ```{r eval=FALSE}
-install.packages("leaflet")
+leaflet() %>%
+  addProviderTiles(provider = 'OpenStreetMap') %>%
+  addGeoJSON(geojson = ecoregions_geoJSON$features) %>%
+  fitBounds(-160,-52,160,60)
 ```
+![map1](tools/img/leaf1_ecoregions.png)
+
 
-Plot data
+
+### Maritime Boundaries (EEZ)
+
+Get Data
+
+```{r}
+eezboundaries_geoJSON <- mr_geojson(key = "MarineRegions:eez_boundaries")
+```
+
+Plot Data
 
 ```{r eval=FALSE}
-library('leaflet')
 leaflet() %>%
   addProviderTiles(provider = 'OpenStreetMap') %>%
-  addGeoJSON(geojson = res1$features) %>%
-  setView(-3.9, 35, zoom = 10)
+  addGeoJSON(geojson = eezboundaries_geoJSON$features) %>%
+  fitBounds(-160,-52,160,60)
 ```
-
-![map](tools/img/leaf1.png)
+![map2](tools/img/leaf1_eezboundaries.png)
 
 ## Shape
 
+### Marine Ecoregions of the World (MEOW)
+
 Get region
 
 ```{r}
-res2 <- mr_shp(key = "MarineRegions:eez_iho_union_v2", maxFeatures = 5)
+ecoregions_shp <- mr_shp(key = "Ecoregions:ecoregions", maxFeatures = 50)
+# check class
+class(ecoregions_shp)
 ```
 
-Get helper library
+Plot data
 
 ```{r eval=FALSE}
-install.packages("leaflet")
+leaflet() %>%
+  addProviderTiles(provider = 'OpenStreetMap') %>%
+  addPolygons(data = ecoregions_shp)
+```
+
+![map4](tools/img/leaf2_ecoregions.png)
+### Maritime Boundaries (EEZ)
+
+Get region
+
+```{r}
+eezboundaries_shp <- mr_shp(key = "MarineRegions:eez_boundaries", maxFeatures = 50)
 ```
 
 Plot data
 
 ```{r eval=FALSE}
-library('leaflet')
 leaflet() %>%
   addProviderTiles(provider = 'OpenStreetMap') %>%
-  addPolygons(data = res2)
+  addPolygons(data = eezboundaries_shp)
 ```
 
-![map2](tools/img/leaf2.png)
+![map5](tools/img/leaf2_eezboundaries.png)
 
 ## Convert to WKT
 
-From geojson
+### Marine Ecoregions of the World (MEOW)
 
-```{r eval=FALSE}
-res3 <- mr_geojson(key = "Morocco:dam")
-mr_as_wkt(res3, fmt = 5)
+From GeoJSON
 
-#> [1] "MULTIPOLYGON (((41.573732 -1.659444, 45.891882 ... cutoff
+```{r eval=FALSE}
+ecoregions_geoJSON <- mr_geojson(key = "Ecoregions:ecoregions")
+ecoregions_wkt1 <- mr_as_wkt(ecoregions_geoJSON, fmt = 2)
+# check class
+class(ecoregions_wkt1) # character
 ```
 
-From shp object (`SpatialPolygonsDataFrame`) or file, both work
+From shp object (using the sf package)
+
 
 ```{r eval=FALSE}
-mr_as_wkt(mr_shp(key = "MarineRegions:eez_iho_union_v2"))
+ecoregions_shp <- mr_shp(key = "Ecoregions:ecoregions", maxFeatures = 50)
+ecoregions_shp_geom <- sf::st_geometry(ecoregions_shp)
+ecoregions_wkt2 <- sf::st_as_text(ecoregions_shp_geom)
+```
+More detailed example using the Black Sea Ecoregion only
+
+```{r, eval=FALSE}
+# subset Black Sea Ecoregion
+ecoregions_blacksea <- subset(ecoregions_shp, ecoregions_shp$ecoregion == "Black Sea")
+## check class
```

Comment:
I think there's no need for a comment here, `class()` is clear already :)

Reply:
I would instead 
```r
ecoregions_blacksea <- subset(ecoregions_shp, ecoregions_shp$ecoregion == "Black Sea")
class(ecoregions_blacksea)
```
with eval = TRUE, it will print
```r
#> [1] "sf"
```

And then, leave a space with the next thing you do

Reply:
and of course no need to hard code the class with comments, RMarkdown will add this!